### PR TITLE
chore/merge-main@9edfed8

### PR DESCRIPTION
**Pull request by julien-deramond**
### Commits from Bootstrap

- [x] https://github.com/twbs/bootstrap/commit/bcfc8d0d633e23dc90eae79464c777515cfcfc97
- [x] https://github.com/twbs/bootstrap/commit/74184f8c61529fbe0b627772a23cd0aa55c99440
- [x] https://github.com/twbs/bootstrap/commit/6e55fa929797a2d1d5250e1630615f515c45d0db
- [x] https://github.com/twbs/bootstrap/commit/9cee77919d801a31a639d12371225d85653212b4
- [x] https://github.com/twbs/bootstrap/commit/d6a38b347fdd4c7f666a3b38f62904bc7f9ecb8d
- [x] https://github.com/twbs/bootstrap/commit/b5ee6694c151fedaa5725bf38451c6717c715e86
- [x] https://github.com/twbs/bootstrap/commit/a1b1e43ddc32e8aba848968023ca0e4e11fb3a55
- [x] https://github.com/twbs/bootstrap/commit/4600a2540463b9ea5a3c505de92d1c0252c01fd5
- [x] https://github.com/twbs/bootstrap/commit/3bd57564140ef3953582a80e8f1084d81bdb5977
- [x] https://github.com/twbs/bootstrap/commit/9af1232c65cd0d63565cd9c8d36b91ef265b26cb
- [x] https://github.com/twbs/bootstrap/commit/aedd7fb9de38efbf6bb4a5443aa870d814df8c55
- [x] https://github.com/twbs/bootstrap/commit/6f65df4faea2694840572626f8a02f4399bd0dca
- [x] https://github.com/twbs/bootstrap/commit/d0117a17d807df05dfd6f6ce72a0d63f081ca529
- [x] https://github.com/twbs/bootstrap/commit/636bb496c93ead07809dc3414e31af3a609482df
- [x] https://github.com/twbs/bootstrap/commit/2be9cd59bae8c864f6d2cfb03c535e96f00160ca
- [x] https://github.com/twbs/bootstrap/commit/1510c25e9be4d8f7f2ba23622bfdfd9afebb6598
- [x] https://github.com/twbs/bootstrap/commit/0173f7a9df22017e3ae75b11220af8ca5880eda6
- [x] https://github.com/twbs/bootstrap/commit/37e2e7e1249b0e514d6d07e3ee1bc01dbdc42c8d
- [x] https://github.com/twbs/bootstrap/commit/27f20257ebd092b2e5d11a62de0cd5e9cf5a9949
- [x] https://github.com/twbs/bootstrap/commit/2b46842af908fe54bd7f7982ed07c7cd07bc934a
- ~~https://github.com/twbs/bootstrap/commit/8da10bb1be424dd42cf5944f124fc8f3245a0cfe~~: we already have a PR template based on this one
- [x] https://github.com/twbs/bootstrap/commit/24f6e2764a557afdf0cabfc28067fe1078e0a0ee
- [x] https://github.com/twbs/bootstrap/commit/eff658edb7d8808b028d4b15de10104d89883df5
- [x] https://github.com/twbs/bootstrap/commit/2b65149e02ed45f9b259a407a335e6e8defad611
- [x] https://github.com/twbs/bootstrap/commit/e77367dddc1a523b6a7197735dbef7cd63d62fe9
- [x] https://github.com/twbs/bootstrap/commit/597c4023141dc48868889b85676b2d7269411d23
- [x] https://github.com/twbs/bootstrap/commit/80482af57b2ac1203e9436398e0153476907f9b4
- [x] https://github.com/twbs/bootstrap/commit/f84d82ada02084cc6567b7ff60f25650084b8bea
- [x] https://github.com/twbs/bootstrap/commit/abb1cf529fb8dfb8e66e44506eb32f8a9e0e0ee2
- [x] https://github.com/twbs/bootstrap/commit/da0fe8c8b2e969bcf620a9ef1c1ca4f2b8554ec6
- [x] https://github.com/twbs/bootstrap/commit/812f891bfca0697cb2ec628700b7bbadbd8bf93a
- [x] https://github.com/twbs/bootstrap/commit/0a5f6e078c9cb9ecca9ab90b3984c5f792a11bae
- [x] https://github.com/twbs/bootstrap/commit/d49d8ce583f6f12552f122122a99063738656cd9
- [x] https://github.com/twbs/bootstrap/commit/9edfed8a5e44d438119627ebfa4ebee1bca01c33

### Commits that need to be explained in the migration guide

- [x] https://github.com/twbs/bootstrap/commit/0a5f6e078c9cb9ecca9ab90b3984c5f792a11bae

### CODE REVIEW

**Review by louismaximepiton — CHANGES_REQUESTED** [review]
LGTM, one thing at least to tackle before merging

File: site/content/docs/5.2/customize/color.md [hunk 1]
```diff
@@ -98,9 +98,9 @@ Color palette is used throughout Boosted in multiple ways, either in components
       </svg>
       <figcaption class="py-1">
         <strong>{{ $color.name }}</strong>
-        <br/>
```
[the remark this answers is not in the document]
Comment (julien-deramond) [reply site/content/docs/5.2/customize/color.md:101]: TBH I only fixed what was an issue when I ran `npm run docs` 😇

File: js/tests/visual/tooltip.html [hunk 2]
```diff
@@ -66,42 +66,73 @@ <h1>Tooltip <small>Bootstrap Visual Test</small></h1>
...
+
+    <script>
+      /* global boosted: false */
+
+      new boosted.Tooltip('#wrapperTriggeredBySelector', {
+        animation: false,
+        selector: '.bs-dynamic-tooltip'
+      })
+
+      /* eslint-disable no-unused-vars */
```
[the remark this answers is not in the document]
Comment (julien-deramond) [reply js/tests/visual/tooltip.html:127]: Yes: `127:16  error  'duplicateButtons' is defined but never used  no-unused-vars`

**Review by louismaximepiton — APPROVED** [review]
LGTM!

**Pull request merged**